src/hex.cpp

```diff
@@ -73,28 +73,31 @@ namespace libtorrent
 	extern char const hex_chars[];
 
 	char const hex_chars[] = "0123456789abcdef";
+	static void to_hex(char const* in, size_t const len, char* out)
```

Comment:
instead of ``static``, please put the function in the anonymous namespace (you can move it up a few lines).